test/release/examples/primers/iterators.chpl

```diff
@@ -2,48 +2,40 @@
    Iterators Primer
 
    This primer contains several examples of iterators:
-     an iterator to generate the Fibonacci numbers,
-     an iterator defined by multiple loops
-     and a recursive iterator over a tree.
+     * an iterator to generate the Fibonacci numbers,
+     * an iterator defined by multiple loops
+     * and a recursive iterator over a tree.
 
    It also contains examples of the two kinds of parallel iteration:
-    data-parallel (forall), and
-    task-parallel (coforall).
+     * data-parallel (``forall``), and
+     * task-parallel (``coforall``).
 */
 
 //
-// fibonacci - generates the first n Fibonacci numbers
+// fibonacci
```

Comment:
Applies to all sections:

We've been adding reference labels to section titles to support potential cross-referencing in the future.

e.g.

```
/*
.. primers-iterators-fibonacci:

fibonacci
---------
*/
```